src/sensors/scripts/segmented_image.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+# Example ROS node for publishing AirSim images.
+
+import setup_path
```

Comment:
What's this import for?

Reply:
not sure tbh, I must have copied it over when I copied the original camera file, I'll remove it and see if it makes any difference

Reply:
https://github.com/microsoft/AirSim/issues/2797
We can remove it, the old group used it to force the local airsim file to be used.